jvm/docker-entrypoint.sh

```diff
@@ -18,7 +21,7 @@ if [ -n "$SURVIVOR_RATIO" ]; then
 	JAVA_OPTS="$JAVA_OPTS -XX:SurvivorRatio=${SURVIVOR_RATIO}"
 fi
 if [ -n "$PARALLEL_GC_THREADS" ]; then
-	JAVA_OPTS="$JAVA_OPTS -XX:ParallelGCThreads=${PARALLEL_GC_THREADS}"
+	JAVA_OPTS="$JAVA_OPTS -XX:ParallelGCThreads=${PARALLEL_GC_THREADS}" # unused unless we set -XX:+UseParallelGC
```

Comment:
...then add a condition to the `if` statement?

Reply:
The condition would be "if $GC_COLLECTOR is G1 or parallel or empty", yes?

Reply:
Won't the JVM just ignore it if a different algorithm is specified?

Reply:
I think this is just a simple shell so, `if [ -n "$PARALLEL_GC_THREADS" -a "GC_COLLECTOR" == "Parallel" ]; then`

Reply:
The flag also applies to the G1: https://docs.oracle.com/javase/8/docs/technotes/guides/vm/gctuning/g1_gc_tuning.html#important_defaults

And if we're defaulting to G1 when GC_COLLECTOR isn't set, we'd want to include that as well.

Reply:
I think I'd like to leave this out since a) it's ignored if not relevant to the current collector and b) the final test statement is messy and I can't seem to get `sh -c` to accept it no matter how I adjust it. Do you feel strongly about changing this?

Reply:
I would remove the comment then, since it is misleading.